samples/cpp/tutorial_code/videoio/orbbec_astra/orbbec_astra.cpp

```diff
@@ -138,56 +135,96 @@ int main()
     });
     //! [Read streams]
 
-    while (true)
+    //! [Pair frames]
+    std::list<std::pair<Frame, Frame>> dcFrames;
+    std::mutex dcFramesMtx;
+    std::condition_variable dcReady;
+
+    // Pair depth and color frames
+    std::thread framePairing([&]
```

Comment:
The thread does not do useful work and is not mentioned in tutorial code. I think that frames sync and UI render in main is enough for tutorial.

Reply:
Done